File: TypeScript/Cryptography/get-sha3-256-of-string.ts
import * as crypto from "crypto"

function main(): void {
    // Get the sha3-256 of a string
    console.log(getSHA3256OfString("Hello World!"))
}

main()

// Get the sha3-256 of a string
function getSHA3256OfString(givenValue: string): string {
    return crypto.createHash("sha3-256").update(givenValue).digest("hex")
}

File: Python/Cryptography/get_sha_3_256_of_string.py
import hashlib


# Get the sha-3-256 of the string.
def get_sha_3_256_of_string(provided_string):
    return hashlib.sha3_256(provided_string.encode()).hexdigest()


def main():
    # Get the sha-3-256 of the string.
    print(get_sha_3_256_of_string("Hello World!"))


main()
